CHANGELOG.md

```diff
@@ -8,6 +8,10 @@ Versioning](https://semver.org/spec/v2.0.0.html).
 
 ## [Unreleased]
 
+### Changed
+
+* Optimized frontend build in our official Docker image
```

Comment:
why a `*` ? We use `-` everywhere ?

Reply:
Good catch! I used vim and not VSCode that automatically lint my markdown sources :wink: